### PR TITLE
SNOW-683718: Bumped up PythonConnector PATCH version from 2.8.0 to 2.8.1

### PR DESCRIPTION
**Pull request by sfc-gh-kdama**
@noreview - This is an automated process. No review is required

Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am modifying authorization mechanisms
   - [ ] I am adding new credentials
   - [ ] I am modifying OCSP code
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

File: DESCRIPTION.md [hunk 1]
```diff
@@ -10,6 +10,13 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 - v2.8.0(September 27,2022)
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:11]:
```suggestion
- v2.8.1(October 30,2022)
```

File: DESCRIPTION.md [hunk 2]
```diff
@@ -10,6 +10,13 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
...
 
+   - Bumped cryptography dependency from <37.0.0 to <39.0.0
+   - Fixed a bug where write_pandas wouldn't write an empty DataFrame to Snowflake
+   - When closing connection async query status checking is now parallelized
+   - Fixed a bug where test logging would be enabled on Jenkins workers in non-Snowflake Jenkins machines
+
+- v2.8.1(October 30,2022)
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:18]:
```suggestion
- v2.8.0(September 27,2022)
```

**Review by sfc-gh-mkeller — CHANGES_REQUESTED** [review]

**Review by sfc-gh-mkeller — APPROVED** [review]
:ship: :ship:

**Comment by sfc-gh-jdu** [comment]
Please mention the version bump for pandas as well. https://github.com/snowflakedb/snowflake-connector-python/commit/e36802e8da82225c1460871e94fcf60c8e247123

**Pull request merged**